packages/manager/apps/bmc-backup-agent-baremetal/package.json

```diff
@@ -32,26 +35,30 @@
     "@ovhcloud/ods-components": "~18.6.2",
     "@ovhcloud/ods-themes": "~18.6.2",
     "@tanstack/react-query": "5.51.21",
+    "@tanstack/react-virtual": "^3.10.9",
```

Comment:
I might be missing something with modules and dependencies, but it looks weird to see this here. Do you use it in the project without outside of the datagrid component ?